sdk/communication/AzureCommunicationChat/Source/NotificationUtils/IdentityUtil.swift

```diff
@@ -70,13 +70,3 @@ func isGcch(id: String) -> Bool {
     let gcchAcsUserPrefix = "gcch-acs:"
     return (id.starts(with: gcchTeamsUserPrefix) || id.starts(with: gcchAcsUserPrefix))
 }
-
-/// Parses out the id/phone number portion of a user id.
-/// - Parameters:
-///   - id: The string id.
-///   - prefix: The id prefix.
-/// - Returns: The part of the id after the prefix that corresponds to the user id or phone number of a user.
-private func parse(id: String, prefix: String) -> String {
-    let index = id.index(id.startIndex, offsetBy: prefix.count)
-    return String(id.suffix(from: index))
-}
```

Comment:
Why is this being removed?

Reply:
It's no longer being used, unless we want to bring everything back from https://github.com/Azure/azure-sdk-for-ios/pull/1468/files#diff-be387cd2134ba545b2160012a828b3b8c544eb948c33dc0228d57bd698a28689, including `getIdentifier(from:)`?

Reply:
Is this something that will be readded when the references to `cloudEnviroment` are updated later on? If so, we should probably just leave this in. It's private after all.

Reply:
No, we'll be utilizing Common's `createCommunicationIdentifier(fromRawId:)` instead. Difference between the 2 versions is the support for MicrosoftBotIdentifier in `1.2.0-beta.1`

Reply:
Okay, so this has been dead code already then?